index.js

```diff
@@ -1,15 +1,14 @@
-const url = require('url')
 const isIp = require('is-ip')
 const Multiaddr = require('multiaddr')
 
 /**
  * Convert a URI to a multiaddr
  *
- *        http://foobar.com => /dns4/foobar.com/https
- *  https://foobar.com:8080 => /dns4/foobar.com/tcp/8080/https
- *          ws://foobar.com => /dns4/foobar.com/ws
- *   https://127.0.0.1:8080 => /ip4/127.0.0.1/tcp/8080/https
- *        http://[::1]:8080 => /ip6/::1/tcp/8080/http
+ *        http://foobar.com => /dns4/foobar.com/tcp/80/http
+ *       https://foobar.com => /dns4/foobar.com/tcp/443/tls/http
+ *  https://foobar.com:5001 => /dns4/foobar.com/tcp/5001/tls/http
+ *   https://127.0.0.1:8080 => /ip4/127.0.0.1/tcp/8080/tls/http
```

Comment:
(same note as above)
```suggestion
 *   https://127.0.0.1:8080 => /ip4/127.0.0.1/tcp/8080/https
```